dockstore-client/src/main/java/io/dockstore/client/cli/ArgumentUtility.java

```diff
@@ -79,7 +88,14 @@ public static void exceptionMessage(Exception exception, String message, int exi
         if (!"".equals(message)) {
             err(message);
         }
-        err(ExceptionUtils.getStackTrace(exception));
+
+        if (LOG.isInfoEnabled()) {
+            err(ExceptionUtils.getStackTrace(exception));
+        } else if (exception.getMessage() != null && !"".equals(exception.getMessage())) {
```

Comment:
`StringUtils.isNotEmpty` would make this clearer: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotEmpty-java.lang.CharSequence-

Reply:
Thanks, I have now done this.